### PR TITLE
Remove checkout and release only on tag with version

### PR DESCRIPTION
**Pull request by locustbaby**
Remove checkout and release only on tag with version
## Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. `[mychartname]`)
- [ ] PR only contains changes for one chart

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -5,6 +5,8 @@ on:
   push:
     branches:
       - master
+    tags:
```
Comment (LoveEachDay) [inline .github/workflows/release.yml:8]: We need to check whether the chart version was bumped when release on tagging.

Comment (locustbaby) [reply .github/workflows/release.yml:8]: Already fixed by add steps `check_version`

**Pull request merged**